src/components/icon/icon.stories.mdx

```diff
@@ -20,6 +20,15 @@ const iconStory = (args) => {
   }
   return template(args);
 };
+const defaultArgs = {
+  name: { type: { name: 'string' } },
+  fallback: { type: { name: 'string' } },
+  inline: { type: { name: 'boolean' } },
+  class: { type: { name: 'string' } },
+  title: { type: { name: 'string' } },
+  muted: { type: { name: 'boolean' } },
+  size: { type: { name: 'string' } },
+};
```

Comment:
I _think_ this can be simplified to the following:
```suggestion
const defaultArgs = {
  name: { type: 'string' },
  fallback: { type: 'string' },
  inline: { type: 'boolean' },
  class: { type: 'string' },
  title: { type: 'string' },
  muted: { type: 'boolean' },
  size: { type: 'string' },
};

```